content/index.js

```diff
@@ -120,6 +120,11 @@ cli
     "don't reuse existing _all-titles.json",
     cli.BOOL
   )
+  .option(
+    "--ignore-titles-cache",
```

Comment:
Nit: This name is confusing to me. It makes me think the opposite of what it actually does, meaning it makes me think that it's going to ignore the cached JSON file and regenerate a new one. What do you think about naming it `--allow-stale-titles` or something like that?

Reply:
I'll change the env var too. Because I think that's a nice and lower cognitive burden.